pkg/helpers.go

```diff
@@ -28,6 +28,14 @@ func ParseID(id string) (string, error) {
 	return parts[1], nil
 }
 
+func ParseRole(id string) (string, error) {
+	parts := strings.Split(id, ":")
+	if len(parts) < 2 {
+		return "", fmt.Errorf("baton-slack: invalid ID: %s", id)
+	}
+	return parts[2], nil
```

Comment:
what if he string is split into just 2 elements? if there is no index 2 (the third element in the array/slice) the connector would panic.

Reply:
resolved